File: lib-connector/src/main/java/com/uis/connector/comm/TypeParam.java
package com.uis.connector.comm;

import java.lang.reflect.Type;

/**
 * 获取范性Type
 * @author uis on 2018/4/27.
 */
@Deprecated
public class TypeParam<T> {
    public Type getType(){
        return TypeConvert.convert(this);
    }
}

File: assignor/src/main/java/com/uis/assignor/utils/TypeParam.kt
/*
 * Copyright (c) 2019 by uis
 * Author: uis
 * Github: https://github.com/luiing
 */

package com.uis.assignor.utils

import java.lang.reflect.Type

/**
 * @autho uis
 * @date 2019-06-12
 * @github https://github.com/luiing
 */
open class TypeParam<T> {
    fun getType() :Type? = TypeConvert.convert(this)
}